karton/classifier/classifier.py

```diff
@@ -300,6 +300,27 @@ def zip_has_mac_app() -> bool:
             )
             return sample_class
 
+        # Various graphics/image file formats
+        image_assoc = {
+            "gif": ["GIF image data"],
+            "jpg": ["JPEG image data"],
+            "png": ["PNG image data"],
+        }
+
+        for ext, patterns in image_assoc.items():
+            if any(pattern in magic for pattern in patterns):
+                sample_class.update({"kind": "dump", "extension": ext})
```

Comment:
I don't think we should use the kind `dump` for image files. That kind is already reserved for memory dumps coming in from analysed samples. Perhaps `misc` would be more appropriate?